git-link.el

```diff
@@ -394,6 +394,10 @@ return (FILENAME . REVISION) otherwise nil."
                     "\\1/_git/"
                     path)))
 
+      ;; For Savannah
+      (when (and (string-match "gnu\\.org\\'" host)
```

Comment:
Is there a benefit to matching the 2nd-level domain vs the complete domain? 

The downside which, I suppose is a valid use case is, it matches for something that's not savannah. Though with the path portion I suppose this does decrease the odds of this happening.

Reply:
Fixed.